src/lib/Parameters.svelte

```diff
@@ -9,10 +9,15 @@
 	export let verticalGap = 30;
 	export let lineGap = 5;
 	export let straightLength = 0;
+	export let endpointCorrection = 0;
 	export let alignment: Alignment = 'center';
 	export let fontFamily: FontFamily = 'default';
 	export let fontStyle: FontStyle = 'normal';
 	export let fontSize = 15;
+
+	$: lineGap = Math.min(lineGap, verticalGap / 2);
+	$: straightLength = Math.min(straightLength, verticalGap / 2);
+	$: endpointCorrection = lineGap <= 0 || straightLength > 0 ? 0 : Math.min(endpointCorrection, lineGap);
```

Comment:
These lines shouldn’t exist since they should’ve already been handled in `RangeSlider.svelte`, but without them it breaks. Do you have any idea how these lines can be eliminated?

Reply:
It seems that in `RangeSlider.svelte` the max/min values are not reactive or the values are not computed from max/min values, i.e the value is not changed if max/min value changed. I don't remember the details, but it could be intentional to:

1. allow invalid values (if the user really want to) instead of making it impossible to do so.
2. avoid potentially circular dependency.

Anyway it is not perfect in this way, because it works, I will keep these three lines.